sceptre/ukbiobank-opendata/README.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+A project to install, configure and manage the strides AWS account.
+
+## Purpose
+The purpose of this project is to build a self service provisioning
+system using the
+[AWS service catalog](https://docs.aws.amazon.com/servicecatalog/latest/adminguide/introduction.html).
```

Comment:
We don't need to deploy Service Catalog in this new account.

Reply:
removed